src/modules/StorageProposal.rb

```diff
@@ -289,6 +300,10 @@ def GetControlCfg
         tmp = ProductFeatures.GetFeature("partitioning", "proposal_snapshots")
         Ops.set(@cfg_xml, "prop_snapshots", tmp == "" || tmp == true ? true : false)
 
+        # GetBooleanFeature cannot distinguish between missing and false
+        tmp = ProductFeatures.GetFeature("partitioning", "need_swap")
+        Ops.set(@cfg_xml, "need_swap", tmp == "" || tmp == true ? true : false)
```

Comment:
Note: If you add support for a completely new option you need to define it in the [schema file](https://github.com/yast/yast-installation-control/blob/master/control/control.rnc#L440) otherwise the control files will not validate.